### ISSUE TITLE
DRC inconsistent with keepout detection.

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#2436]
_From [bitsybof...@gmail.com](https://code.google.com/u/112850805511005419452/) on February 22, 2013 02:49:39_

Attached 2 images.  

The first shows the DRC when the keepout is set to 0.254mm, as you can see it's just the off-board markers that fail, excellent so everything else should be MORE than 0.254 away from everything right?  That is, there should be nothing closer to anything else (on a different net) than 0.254mm on that board, right?

But the second images shows the DRC on the exact same board, with the minimum keepout reduced to 0.200mm.  Suddenly we get failures between a couple of holes and traces, meaning that some stuff the DRC now thinks is closer than 0.200mm to other stuff.  

That doesn't make sense, at 0.254 the DRC thinks everything is more than 0.254, at 0.200 the DRC thinks some stuff is closer than 0.200.

If the DRC passes something at a minimum keepout of 0.254, then it must _by definition_ pass at any smaller minimum keepout, shouldn't it?

In short, I don't think the DRC is checking the keepout very well.

Have attached the fzz too in case you want to try.

**Attachment:** [keepout-0.254.png keepout-0.200.png drill.6.fzz](http://code.google.com/p/fritzing/issues/detail?id=2438)

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=2438_
